idkit/src/store/idkit.ts

```diff
@@ -49,9 +55,32 @@ const useIDKitStore = create<IDKitStore>()((set, get) => ({
 	verifyCallbacks: {},
 	successCallbacks: {},
 	stringifiedActionId: '',
-	stage: IDKITStage.ENTER_PHONE,
+	methods: ['orb', 'phone'],
```

Comment:
We can't turn it into an enum since then TS will complain when you pass the value as a string (this is fixed in the next Typescript beta but not yet merged in main)